src/undocumented.md

```diff
@@ -14,20 +14,14 @@ to shrink!
   specified.
 - [Flexible target specification] - Some---but not all---flags are documented
   in [Conditional compilation]
-- [Require parentheses for chained comparisons]
```

Comment:
Is this supposed to be part of a separate PR?

Reply:
no: that feature is already documented [here](https://doc.rust-lang.org/reference/expressions/operator-expr.html#comparison-operators), so I'm just cleaning this list up.